README.md

```diff
@@ -1 +1,17 @@
-# Hello_Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Hello Microverse
+
+## Built With
+
+- html
+- css
+
+
+## Authors
+
+👤 **Roland-Ntwali**
+
+- GitHub: [@Roland-Ntwali ](https://github.com/Roland-Ntwali)
+- Twitter: [@_Ntwali](https://twitter.com/_Ntwali)
+- LinkedIn: [@Roland Ntwali](https://www.linkedin.com/in/roland-ntwali-11b16617b/)
```

Comment:
- [ ] As requested by the previous reviewer please add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but replace the placeholder text with information about your project.